timeplus/client.go

```diff
@@ -15,6 +16,18 @@ import (
 
 const TimeFormat = "2006-01-02 15:04:05.000"
 const APIVersion = "v1beta1"
+const APIV2Version = "v1beta2"
+
+type QueryEvent map[string]any
+type MetricsEvent map[string]any
+type DataEvent [][]any
```

Comment:
I think we should create an alias for our `Data` even though it is just `[]any`. In Neutron there are just too many `[]any` so it is hard to tell which one represent the data itself

Reply:
Yes, for a complete SDK, like we discussed, we need do type mapping for each cell in the row.  this is more for internal usage for now, let's plan it when we do need a SDK for this.